BenchmarkVCFs/CompareBenchmarks.wdl

```diff
@@ -388,7 +394,7 @@ def main(sample_ids, configurations, summaries, stratifiers, order_of_samples, o
         sample_data['configuration'] = configurations[i]
         samples_data.append(sample_data)
     data = pd.concat(samples_data)
-    data = data.fillna({'Stratifier': 'all'})
+    data = data.fillna({'Stratifier': 'WholeGenome'})
```

Comment:
This is getting very technical but I've run this pipeline on WES data in the past and then this label wouldn't be very accurate. Maybe AllRegions or something like that?